_includes/modern-appendix.md

```diff
@@ -413,6 +413,29 @@ Sent as a reply to the {% message AWAY %} command, this lets the client know tha
 
 Sent as a reply to the {% message AWAY %} command, this lets the client know that they are set as being away. The text used in the last param of this message may vary.
 
+{% numericheader RPL_WHOREPLY %}
+
+      "<client> <channel> <username> <host> <server> <nick> <flags> :<hopcount> <realname>"
```

Comment:
irc2 actually uses this syntax:

```
"<client> <channel> <username> <host> <server> <nick> <flags> :<hopcount> <SID> <realname>"
```

But I'm tempted to ignore this, it won't significantly break clients, and irc2 is the only one to do this.

Reply:
Right. This sounds painful to detect and handle in clients regardless.